packages/ava/README.md

```diff
@@ -0,0 +1,148 @@
+near-runner + Jest
```

Comment:
```suggestion
near-runner + AVA
```

Note that the team [cares about how it’s written](https://github.com/avajs/ava#how-is-the-name-written-and-pronounced) and I think we should call projects what their maintainers want them to be called.

I only suggested a change to this line, but the whole README needs to be updated before we merge & release this. I can help with that, if you’d like.